### ISSUE TITLE
[FEATURE REQUEST] Add option for automatically deleting images after successful upload

### ISSUE THREAD

**Issue by FReichelt** [simulot/immich-go#187]
Hello Immich-Go Team,

I'm proposing a feature to automatically delete images from the device after they've been successfully uploaded to the server. This would reduce useless rechecking of files when rerunning.

Thank you guys in advance :)

**Comment by FReichelt** [comment]
Is there any reason why the delete option has been disabled with this [commit](https://github.com/simulot/immich-go/commit/4a7e2aec943e862219709fc3a5d7196a9bd14619)?

**Comment by simulot** [comment]
Yes... immich-go does some bet on photos duplicates and best resolution. None of this is 100% reliable and I prefer to leave the user to do the cleaning after having checked the import result.

So, after the import process, some photos haven't been uploaded because of program decision.
Should we delete those one too?

**Comment by FReichelt** [comment]
> Yes... immich-go does some bet on photos duplicates and best resolution. None of this is 100% reliable and I prefer to leave the user to do the cleaning after having checked the import result.
> 
> So, after the import process, some photos haven't been uploaded because of program decision. Should we delete those one too?

For me deleting every successfully uploaded would do enough, I believe some way of checking how the program decided would be a great solution too. For example, simply creating a text file or json which lists how each file was handled.

**Comment by simulot** [comment]
You can activate logs for having the details

**Comment by FReichelt** [comment]
> You can activate logs for having the details

Yes, that's true, but for the average immich-go user it is still very difficult to process the data automatically.

**Comment by p166** [comment]
I really miss this feature!

**Comment by CumpsD** [comment]
I would love it to have a flag where I can say to delete every successfully uploaded file. Then I know the remaining files need to be dealt with manually.
